microsoft-service-bus/2.2/modules/ROOT/pages/index.adoc

```diff
@@ -259,18 +282,18 @@ To enable the SSL checks, follow these steps to import the certificate:
 . Use the PowerShell cmdlet https://msdn.microsoft.com/library/azure/jj248762%28v=azure.10%29.aspx[Get-SBAutoGeneratedCA] to download the certificate locally in the machine running the Windows Service Bus. +
 For this example, the certificate file is exported to `%temp%\AutoGeneratedCA.cer`.
 . Running as Administrator, go to `%programfiles%\Java\jre7`. +
-you must be running as Administrator to perform a certificate import with `Keytool.exe`. 
-. Verify that the `bin\keytool.exe` tool and `lib\security\cacerts` exists. 
+you must be running as Administrator to perform a certificate import with `Keytool.exe`.
+. Verify that the `bin\keytool.exe` tool and `lib\security\cacerts` exists.
 . Enter the following command: +
 `bin\keytool.exe –list –keystore lib\security\cacerts`
 . Import the autogenerated Service Bus certificate by running the following command: +  `bin\keytool.exe –importcert –alias AppServerGeneratedSBCA –file %temp%\AutoGeneratedCA.cer –keystore lib\security\cacerts –v`
 . Enter the password when prompted (the default is “changeit”).  +
-If you don't know the password, you can't perform the import.  
+If you don't know the password, you can't perform the import.
 . When the tool asks you whether to trust the certificate, enter `Y` (Yes).
 
 == Using Restricted Access Policies
 
-If you have restricted access to your resources, such as having a security policy with permissions at the resource level, the connector cannot perform the connectivity test when it starts because the connectivity test targets the root level of your namespace, which might be forbidden due to the customized policy applied to the `shared access key`. For these scenarios, skip the connectivity test with the configuration option available for this purpose, otherwise, the connector fails to start.  
+If you have restricted access to your resources, such as having a security policy with permissions at the resource level, the connector cannot perform the connectivity test when it starts because the connectivity test targets the root level of your namespace, which might be forbidden due to the customized policy applied to the `shared access key`. For these scenarios, skip the connectivity test with the configuration option available for this purpose, otherwise, the connector fails to start.
```

Comment:
First sentence is run-on.  In the second sentence, what is the available configuration option called?

Reply:
Out of scope  - I didn't this. I will fix the run-on though.